lib/card.rb

```diff
@@ -0,0 +1,18 @@
+class Card
+
+  attr_reader :id
+
+  def initialize(id, attributes)
+    @id = id
+    @attributes = attributes
+  end
+
+  def method_missing(method_name, *args)
```

Comment:
I wanted to avoid explicitly defining the attributes in the card itself, which is probably emulating poodir a _little_ too much without considering the costs associated with `method_missing`. I already removed the `method_missing` override in the other feature branch since it didn't make much sense to me later on.

I toyed with the idea of making `Card` inherit from `OpenStruct` to make it explicit that card is just a data container. Thoughts?